### PR TITLE
Re-enable new run command

### PR DESCRIPTION
**Pull request by henneberger**

### CODE REVIEW

File: sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/SimpleSqrlIT.java [hunk 1]
```diff
@@ -1,160 +1,160 @@
...
+            "PGPORT", testDatabase.getMappedPort(PostgreSQLContainer.POSTGRESQL_PORT).toString(),
+            "PGPASSWORD", testDatabase.getPassword(),
+            "PGDATABASE", testDatabase.getDatabaseName(),
+            "PROPERTIES_BOOTSTRAP_SERVERS", container.getBootstrapServers()
+        ));
+    TableResult run1 = run.run(false);
+
+    int count = 10;
+    postGraphQLMutations(count);
+    getGraphqlQuery();
```
Comment (mbroecheler) [inline sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/SimpleSqrlIT.java:67]: you should pass the expected record count in there as well.

File: sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/SimpleSqrlIT.java [hunk 2]
```diff
@@ -1,160 +1,160 @@
...
+      // Navigate through the JSON to get the 'MyTable' data
+      JsonNode myTableData = jsonResponse.path("data").path("MyTable");
+
+      if (!myTableData.isArray()) {
+        continue;
+      }
+
+      // Check the number of records
+      recordCount = myTableData.size();
+      if (recordCount >= expectedRecordCount) {
```
Comment (mbroecheler) [inline sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/SimpleSqrlIT.java:137]: shouldn't this be equal

File: sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/UdfIT.java [hunk 3]
```diff
@@ -1,195 +1,197 @@
...
+      Thread.sleep(1000);
+    }
+
+    if (recordCount < expectedRecordCount) {
+      fail("Failed to retrieve 10 records within the timeout period. Only got " + recordCount);
+    }
+  }
+
+  public void buildUdf() {
+    File projectDir = new File("/Users/henneberger/sqrl/sqrl-testing/sqrl-integration-tests/src/test/resources/udf/myjavafunction"); // Update with your project path
```
Comment (mbroecheler) [inline sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/UdfIT.java:163]: make this relative or look up system.dir

File: sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/sensors-full--package.txt [hunk 4]
```diff
@@ -170,13 +163,9 @@ LogicalTableScan(table=[[sensors_2]])
...
-    "EXECUTE STATEMENT SET BEGIN\nINSERT INTO `hightemp_1`\n(SELECT *\n FROM `table$3`)\n;\nINSERT INTO `lasthour_1`\n (SELECT *\n  FROM `table$7`)\n ;\n INSERT INTO `machine_1`\n  (SELECT *\n   FROM `table$11`)\n  ;\n  INSERT INTO `secreading_1`\n   (SELECT *\n    FROM `table$13`)\n   ;\n   INSERT INTO `sensorreading_2`\n    (SELECT *\n     FROM `table$14`)\n    ;\n    INSERT INTO `sensors_3`\n     (SELECT *\n      FROM `table$15`)\n     ;\n     INSERT INTO `hightemp_1_1`\n      (SELECT *\n       FROM `table$18`)\n      ;\n      END;"
+    "EXECUTE STATEMENT SET BEGIN\nINSERT INTO `hightemp_1`\n(SELECT *\n FROM `table$3`)\n;\nINSERT INTO `lasthour_1`\n (SELECT *\n  FROM `table$7`)\n ;\n INSERT INTO `machine_1`\n  (SELECT *\n   FROM `table$11`)\n  ;\n  INSERT INTO `secreading_1`\n   (SELECT *\n    FROM `table$13`)\n   ;\n   INSERT INTO `sensorreading_2`\n    (SELECT *\n     FROM `table$14`)\n    ;\n    INSERT INTO `sensors_3`\n     (SELECT *\n      FROM `table$15`)\n     ;\n     END;"
   ],
   "connectors" : [
-    "print",
```
Comment (mbroecheler) [inline sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/sensors-full--package.txt:179]: Why did you have to remove the export to print from this test?

File: sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot [hunk 5]
```diff
@@ -1 +1 @@
-{"data":{"MyTable":[{"id":101.0,"hello":"hello world"},{"id":99.0,"hello":"hello world"},{"id":98.0,"hello":"hello world"},{"id":97.0,"hello":"hello world"},{"id":96.0,"hello":"hello world"},{"id":95.0,"hello":"hello world"},{"id":95.0,"hello":"hello world"},{"id":94.0,"hello":"hello world"},{"id":93.0,"hello":"hello world"},{"id":92.0,"hello":"hello world"}]}}
\ No newline at end of file
+{"data":{"MyTable1":[{"id":101.0,"hello":"hello world"},{"id":29.0,"hello":"hello world"},{"id":85.0,"hello":"hello world"},{"id":14.0,"hello":"hello world"},{"id":85.0,"hello":"hello world"},{"id":35.0,"hello":"hello world"},{"id":95.0,"hello":"hello world"},{"id":30.0,"hello":"hello world"},{"id":34.0,"hello":"hello world"},{"id":58.0,"hello":"hello world"}]}}
```
Comment (mbroecheler) [inline sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot:1]: the data looks wrong

File: sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/postgres-query.snapshot [hunk 6]
```diff
@@ -1 +1 @@
-{"data":{"MyTable2":[{"id":1.0,"hello":"hello world"},{"id":1.0,"hello":"hello world"},{"id":2.0,"hello":"hello world"},{"id":3.0,"hello":"hello world"},{"id":3.0,"hello":"hello world"},{"id":4.0,"hello":"hello world"},{"id":5.0,"hello":"hello world"},{"id":6.0,"hello":"hello world"},{"id":7.0,"hello":"hello world"},{"id":8.0,"hello":"hello world"}]}}
\ No newline at end of file
+{"data":{"MyTable2":[{"id":101.0,"hello":"hello world"},{"id":29.0,"hello":"hello world"},{"id":85.0,"hello":"hello world"},{"id":14.0,"hello":"hello world"},{"id":85.0,"hello":"hello world"},{"id":35.0,"hello":"hello world"},{"id":95.0,"hello":"hello world"},{"id":30.0,"hello":"hello world"},{"id":34.0,"hello":"hello world"},{"id":58.0,"hello":"hello world"}]}}
```
Comment (mbroecheler) [inline sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/postgres-query.snapshot:1]: the data looks wrong

**Review by mbroecheler — COMMENTED** [review]
Overall this looks good to me and the additional test coverage is awesome.
I didn't look at DataSQRLNewRun too closely.

File: sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot (hunk 5 above)
Comment (mbroecheler) [reply sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot:1]: looks like we lost the default order?

File: sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/sensors-full--package.txt (hunk 4 above)
Comment (henneberger) [reply sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/sensors-full--package.txt:179]: the thousandish print statements were making the job slow

File: sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/flink/DagPlannerIT.java [hunk 7]
```diff
@@ -2,6 +2,7 @@
 //
```
Comment (mbroecheler) [inline sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/flink/DagPlannerIT.java:2]: It seems we lost this test somewhere along the way. Why?

File: sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot (hunk 5 above)
Comment (henneberger) [reply sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot:1]: does it? the previous queries had an order statement which made it 'look' more correct. ill double check that it is actually iceberg queries?

Comment (mbroecheler) [reply sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot:1]: Yeah, maybe the old result was wrong. I just noticed the difference, i.e. one of them must  be wrong ;-)

Comment (mbroecheler) [reply sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/iceberg-query.snapshot:1]: Actually, this is the order the data is in.

File: sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/sensors-full--package.txt (hunk 4 above)
Comment (mbroecheler) [reply sqrl-testing/sqrl-integration-tests/src/test/resources/snapshots/com/datasqrl/UseCaseCompileTest/sensors-full--package.txt:179]: 👍

File: sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/postgres-query.snapshot (hunk 6 above)
Comment (mbroecheler) [reply sqrl-testing/sqrl-integration-tests/src/test/resources/usecases/duckdb/snapshots-queries/postgres-query.snapshot:1]: this is actually the correct data. The old version was wrong.

**Pull request merged**